extension/js/common/composer.ts

```diff
@@ -1599,7 +1603,7 @@ export class Composer {
       this.debug(`renderPubkeyResult: Setting email to wrong / misspelled in harsh mode: ${email}`);
       $(emailEl).attr('title', 'This email address looks misspelled. Please try again.');
       $(emailEl).addClass("wrong");
-    } else if (contact.pubkey && await Pgp.key.usableButExpired((await openpgp.key.readArmored(contact.pubkey)).keys[0])) {
+    } else if (contact.pubkey && (contact.expiresOn || Number.MAX_SAFE_INTEGER) <= Date.now()) {
```

Comment:
Here you will need to revert the change back, and still actually check the key.

Users will have keys in in the db that were imported before this change, an these keys don't have `exporesOn`, so this check would fail.

Maybe you can do both:
```js
contact.pubkey && ((contact.expiresOn || Number.MAX_SAFE_INTEGER) <= Date.now() || await Pgp.key.usableButExpired((await openpgp.key.readArmored(contact.pubkey)).keys[0]))
```
To skip the potentially expensive check at least sometimes.

Reply:
Yes, agree,
I forgot about that